### PR TITLE
Include app version archive in support bundle

### PR DESCRIPTION
**Pull request by emosbaugh**
Unresolved issues:

1. redactors make tar or tar.gz invalid
2. tar is dropped in a directory deep within the bundle `/default/kotsadm-6b588f457c-jzx4k/kotsadm/tmp/app-version-archive545366197/ethan-kots.tar`

### CODE REVIEW

File: kotsadm/pkg/handlers/troubleshoot.go [hunk 1]
```diff
@@ -861,6 +886,102 @@ func makeVeleroCollectors() []*v1beta1.Collect {
...
+		return nil, errors.Wrapf(err, "failed to walk archive dir %s", archivePath)
+	}
+
+	return &v1beta1.Collect{
+		Copy: &v1beta1.Copy{
+			CollectorMeta: v1beta1.CollectorMeta{
+				CollectorName: fmt.Sprintf("%s-app-version-archive", app.Slug),
+			},
+			Selector: []string{
+				"app=kotsadm", // can we assume this?
```
Comment (emosbaugh) [inline kotsadm/pkg/handlers/troubleshoot.go:976]: this will not work in dev without https://github.com/replicatedhq/kots/pull/917

**Comment by emosbaugh** [comment]
LGTM

File: kotsadm/pkg/handlers/troubleshoot.go [hunk 2]
```diff
@@ -752,3 +778,108 @@ func makeVeleroCollectors() []*troubleshootv1beta2.Collect {
...
+		if err != nil {
+			return err
+		}
+		if archivePath == path {
+			// root dir itself is the first item in the walk
+			return nil
+		}
+
+		trimmedPath := strings.TrimPrefix(path, archivePath)
+		fmt.Printf("+++++trimmedPath:%s\n", trimmedPath)
```
Comment (salahalsaleh) [inline kotsadm/pkg/handlers/troubleshoot.go:834]: forgot to remove this :)

Comment (divolgin) [reply kotsadm/pkg/handlers/troubleshoot.go]: 👍

**Review by salahalsaleh — APPROVED** [review]

**Pull request merged**